src/prefect/blocks/core.py

```diff
@@ -609,15 +616,36 @@ def _define_metadata_on_nested_blocks(
     @classmethod
     @sync_compatible
     @inject_client
-    async def load(cls, name: str, client: "OrionClient" = None):
+    async def load(
+        cls,
+        name: str,
+        skip_validation: Optional[bool] = False,
```

Comment:
Thoughts on `validate: bool = True` instead? I generally try to avoid double negatives. Also, this doesn't look like an `Optional` type — a value of `None` is not allowed.